### PR TITLE
[MIG] crm_timesheet: Migration to 11.0

### PR DESCRIPTION
**Pull request by cubells**
cc @Tecnativa

### CODE REVIEW

File: crm_timesheet/tests/test_account_analytic_line.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import SavepointCase
```
Comment (oihane) [inline crm_timesheet/tests/test_account_analytic_line.py]: Wouldn't it be `odoo.tests.common`
:wink:

Comment (cubells) [reply crm_timesheet/tests/test_account_analytic_line.py]: @oihane sure! 

Done.

**Review by pedrobaeza — APPROVED** [review]

**Review by oihane — APPROVED** [review]
Code looks OK

File: crm_timesheet/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        'crm',
+        'hr_timesheet'
+    ],
+    'data': [
+        'views/crm_lead_view.xml',
+        'views/hr_timesheet_view.xml'
+    ],
+    'author': 'Tecnativa, '
+              'Odoo Community Association (OCA)',
+    'website': 'http://www.tecnativa.com',
```
Comment (chienandalu) [inline crm_timesheet/__manifest__.py]: http://github.com/OCA/hr-timesheet

**Review by chienandalu — APPROVED** [review]
Looks good. Tested on runbot :+1:

**Pull request merged**